third_party/gpus/cuda_configure.bzl

```diff
@@ -824,10 +824,20 @@ def _cuda_autoconf_impl(repository_ctx):
     _create_cuda_repository(repository_ctx)
 
 
+
 cuda_configure = repository_rule(
     implementation = _cuda_autoconf_impl,
-    local = True,
+    environ = [
+        _GCC_HOST_COMPILER_PATH,
+        "TF_NEED_CUDA",
```

Comment:
This one modifies the behaviour in configure script, but does anything else depend on it?